docs/config/README.md

```diff
@@ -3681,11 +3681,12 @@ Starship gets the current Ruby version by running `ruby -v`.
 
 ### Variables
 
-| Variable | Example  | Description                          |
-| -------- | -------- | ------------------------------------ |
-| version  | `v2.5.1` | The version of `ruby`                |
-| symbol   |          | Mirrors the value of option `symbol` |
-| style\*  |          | Mirrors the value of option `style`  |
+| Variable | Example      | Description                                                                |
+| -------- | ------------ | -------------------------------------------------------------------------- |
+| version  | `v2.5.1`     | The version of `ruby`                                                      |
+| symbol   |              | Mirrors the value of option `symbol`                                       |
+| style\*  |              | Mirrors the value of option `style`                                        |
+| gemset   | `2.0.0@test` | Reads the Gemset and its version from the environment variable `GEM_HOME`. |
```

Comment:
I think the gemset and version should be separate, so people can format it however they like it, like using an icon instead of the `@` sign.

Reply:
Ah, so like `gemset-version` and `gemset-name`?

Reply:
As far as I know, gemsets themselves don't have versions, they belong to an installed ruby version[^1]. Gemset `2.5.0@my-gemset`, for instance, is part of the ruby 2.5.0 install.

Here's an example:
<img width="1159" alt="image" src="https://github.com/starship/starship/assets/893971/2bf5e47a-8b33-44f6-b650-90173177916b">

Reply:
I see, so it'd be better off to just grab the name on its own, and use the already-fetched `version` variable and allow the users to format it themselves